webapp/components/button.tsx

```diff
@@ -1,4 +1,9 @@
+import Link from 'next-intl/link'
 import { ComponentProps } from 'react'
+import { isRelativeUrl } from 'utils/url'
+
+const commonCss = `text-ms box-content flex items-center justify-center
+  rounded-lg border border-solid px-3 py-1.5 font-medium leading-5 disabled:opacity-40`
 
 const variants = {
   primary: `border-orange-700/55 from-orange-500 to-orange-600 text-white hover:border-orange-700/70  
```

Comment:
Same here

Reply:
Same as [here](https://github.com/hemilabs/ui-monorepo/pull/522#discussion_r1765570797) - even more here, as it is longer 😅

Reply:
ahh my mistake was just a quick look and I didn't notice